### DISCUSSION TITLE
Unable to edit pages in the outline instance I hosted on Ubuntu vm

### DISCUSSION THREAD

**Discussion by MSVO** [outline/outline#4975] [Self Hosting] [answered]
Hello Outline Self Hosting Community!

I self hosted outline using docker compose on my ubuntu vm by following the guide here https://wiki.generaloutline.com/s/hosting/doc/docker-7pfeLP5a8t . I am able to authenticate to the instance using slack and create pages. However, I am not able to edit content of any page. The cursor does not show up when I click on the content area. One time the cursor did show up and I was able to edit the page, However the content was empty when I try to view the page through incognito tab after making it public. Page title was visible. It seems like content only stayed in Redis or something and didn't go to persisted storage. Please help me debug this issue.

**Comment by apoorv-mishra** [comment, marked as answer]
Somewhat related to https://github.com/outline/outline/discussions/3911.

**Reply by MSVO** [reply]
Thank you @apoorv-mishra for the quick response and pointing me to the right direction. I added the nginx configuration for web sockets as mentioned in #2222 . Issue resolved.
